variables.tf

```diff
@@ -110,6 +110,16 @@ variable "mirror_repositories" {
       description = "Mage-OS mirror fork of https://github.com/magento/zf1."
       teams       = ["continuous-integration"]
     }
+
+    mirror-magento-cloud-patches = {
```

Comment:
I tried to sort the repositories alphabetically - would you mind updating it? :see_no_evil:

Reply:
I think the current naming scheme is `mirror-{{ORIGINAL_REPOSITORY_NAME}}`.